dist/doc/html.md

```diff
@@ -176,6 +176,13 @@ included.
 We use `analytics.js` rather than the newer `gtag.js` as
 [it's faster and supports tasks and plugins](https://github.com/philipwalton/analyticsjs-boilerplate/issues/19#issuecomment-333714370)
 
+Starting with version 8 we, by default, <a href="https://support.google.com/analytics/answer/2763052?hl=en">anonymizse IP addresses</a>.
+By default Google Analytics records the full IP address of a user visiting the site, but
+that full IP address is never available to the Google Analytics property admin.
+By anonymizing the IP address you can make your site more GDPR-compliant as a
+full IP address can be defined as PII (publicly identifiable information.)
```

Comment:
sorry another thing... my fault as this was my comment... but this should actually be personally identifiable information (PII)
REF: https://en.wikipedia.org/wiki/Personal_data